pset-functions.js

```diff
@@ -208,6 +293,21 @@ console.log('Problem 5:')
 
 // Add your code below this line
 
+function doubleLetters(str){
```

Comment:
@rvaswani85 good job getting this to work. Here you've used a nested `for...loop` which works. However, nested logic tends to be harder to read and harder to troubleshoot. Take a look at an [alternative approach taken here](https://github.com/jsd20190730/hw-02-functions-and-iterators-solution/blob/master/pset-functions.js#L300) to see one way to approach this without nesting